crud/src/main/java/com/apirest/crud/dtos/userDtoImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.apirest.crud.dtos;
+
+import com.apirest.crud.model.User;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+@Component
+public class userDtoImpl{
+
+    public UserDto convertToDTO(User user) {

```

Comment:
not required a class for this method, this can be done in controller layer as a private method